{"name": "Ошибка при неверном пароле", "status": "passed", "start": 1751895228080, "stop": 1751895228566, "uuid": "ac073cb3-5edd-4c62-97cc-71f83f7a1d13", "historyId": "0036adbe4717b453c70c5db926a90f89", "testCaseId": "0036adbe4717b453c70c5db926a90f89", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_wrong_password", "labels": [{"name": "epic", "value": "Курьеры"}, {"name": "feature", "value": "Логин курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-3L4TMJM"}, {"name": "thread", "value": "23940-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}